[4.1] jooa11y plugin options

tecpromotion opened the pull request:
Pull Request für Issue #2302 .

### Zusammenfassung der Änderungen

fix strings

### Wo wird der Sprachstring angezeigt / Wie kann getestet werden

backend

tecpromotion committed:
fix #2302

max123kl reviewed (commented):
"Landmark" ist kein Begriff, der mir für mich keine sinnvolle Übersetzung darstellt, besonders nicht im Zusammenhang mit Barrierefreiheit. Das Wort sollte korrekt als "Landmarke" geschrieben werden.

Ich würde hier "Orientierungspunkt" einsetzen.

siehe auch: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/landmark_role
Zitat (übersetzt):
"Ein Orientierungspunkt ist ein wichtiger Teilabschnitt einer Seite. Die Landmark-Rolle ist eine abstrakte Oberklasse für die aria-Rollenwerte für Inhaltsabschnitte, die so wichtig sind, dass die Benutzer wahrscheinlich direkt zu ihnen navigieren möchten."

tecpromotion committed:
fix landmark

@max123kl danke für den Hinweis. Nach einer Recherche passt „landmark role“ jetzt sehr gut mit dem Zusatz role. Dazu findet man dann auch leichter als Benutzer Hinweise im Netz.
